docker-compose.yml

```diff
@@ -1,7 +1,11 @@
+---
 services:
   web:
     restart: always
-    image: ghcr.io/dalibo/explain.dalibo.com:latest
```

Comment:
Actually, images are built and published on `ghrc.io`  via a [github action](https://github.com/dalibo/explain.dalibo.com/commit/4a8b488b8fad04338f210a1c07ed7a8d45df143f).

Reply:
I removed that commit in favor of another one. Now the docker image is pulled from the docker hub registry (which is the default in most situation).